scripts/ci/build-and-test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -eu
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)" # Figure out where the script is running
+. "$SCRIPT_DIR"/../lib/robust-bash.sh
+
+npm i
```

Comment:
This should probably be `npm ci`, not install - but also I would recommend using the full argument names in scripts - it makes them easier to read for people who aren't familiar with the ecosystem.

Reply:
We don't have a committed package lockfile due to this in the ignore file

https://github.com/pact-foundation/jest-pact/blob/f5b48877e4158f128ce47de8dc3b78709176d3af/.gitignore#L19

so ci is no bueno :(

> npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.